chemprop/data/utils.py

```diff
@@ -242,6 +233,19 @@ def get_data(path: str,
             if len(all_smiles) >= max_data_size:
                 break
 
+        atom_features = None
+        atom_descriptors = None
+        if args is not None and args.atom_descriptors is not None:
+            try:
+                descriptors = load_valid_atom_features(atom_descriptors_path, [x[0] for x in all_smiles])
+            except Exception as e:
+                raise ValueError('Failed to load or valid custom atomic descriptors: {}'.format(e))
```

Comment:
Can you switch this to an f-string like below? I just think f-strings are a little easier to read and that way we're consistent with the rest of chemprop.

```suggestion
                raise ValueError(f'Failed to load or valid custom atomic descriptors: {e}')
```